components/brave_news/browser/feed_v2_builder.h

```diff
@@ -88,7 +88,14 @@ class FeedV2Builder : public PublishersController::Observer {
   void OnPublishersUpdated(PublishersController* controller) override;
 
  private:
-  using UpdateCallback = base::OnceCallback<void()>;
+  // FeedGenerator's will be called on a different thread. We guarantee that the
+  // data in |builder| will not be changed while the generator is running.
```

Comment:
optional: can we add one more comment, like

```suggestion
  // data in |builder| will not be changed or invalidated while the generator is running. 
```

so as to let others know that it's safe to have `const&` rather than `WeakPtr`.